packages/Datadog.Unity/Runtime/DatadogSdk.cs

```diff
@@ -73,6 +73,18 @@ public static void Shutdown()
             Instance.ShutdownInstance();
         }
 
+        /// <summary>
+        /// Sets the verbosity level of the SDK. This will affect the amount of logs that the SDK will output.
+        /// </summary>
+        /// <param name="logLevel">The level of SDK verbosity.</param>
+        public void SetSdkVerbosity(CoreLoggerLevel logLevel)
```

Comment:
probably we can omit using `sdk` in the name, both iOS and Android cores don't have this word in the similar method names.

```suggestion
        public void SetVerbosity(CoreLoggerLevel logLevel)
```